README.md

```diff
@@ -67,6 +67,9 @@ yarn start
 
 This will start a watcher that will compile the code, then run a script that creates an ngrok tunnel and starts up the dev server behind it. When the server starts, it automatically updates the Telegram webhook to point at your ngrok tunnel. If you get the console message "MongoDB connection successful..." you're up and running!
 
+## Tests
+We use [Jest](https://jestjs.io/docs/getting-started) for testing. To run the tests, run `npm test`. If you don't have NPM installed locally, you can run the command `docker-compose exec node sh -c \"./node_modules/.bin/jest --verbose\"` instead.
```

Comment:
Instead of asking them to run this command, we could add an npm command to run it for them. Like `test:docker` or something?

Reply:
"npm run test" will run the tests inside docker but you still need to have npm installed locally to do that.

Reply:
Oh wait, that command is what `test` runs. Was the `docker-compose exec node...` command supposed to be something else, or was that not supposed to be in `test`?

Reply:
oh duh

Reply:
Wait, could we somehow allow them to run `npm test` inside the container with `docker-compose`?

Reply:
I see what you're saying. Sure, I changed the command `test` to just run the tests and now `test:docker` will run them via the docker containers. Also, updated the readme to reflect those changes.

Reply:
Nice work man, this works perfectly